tests/test_end_to_end.py

```diff
@@ -89,6 +89,19 @@ def setUp(self):
             schema="string",
         )
 
+    def get_table_name(
```

Comment:
let's rename this as create_and_get_table_name, since there is another function `get_table_name`